src/plugins/requestAuthPlugin.js

```diff
@@ -200,8 +205,8 @@ export const getAuthenticate = authorizeRequest$ => (request, reply) => {
 			request.log(['info', 'auth'], 'Request authenticated');
 		})
 		.subscribe(
-			({ state: { oauth_token } }) => {
-				const credentials = oauth_token;
+			({ state: { MEETUP_MEMBER, oauth_token } }) => {
```

Comment:
technically `MEETUP_MEMBER` will contain the correct value at this point in the request lifecycle, but I'll change it to be more explicit